retroactive/src/ogn.ts

```diff
@@ -370,14 +370,21 @@ const calculateRewards = async () => {
     ),
   ];
 
+  const exclusionList = (fs.readFileSync("./exclusions.csv", "utf-8")).split("\n").map(ethers.utils.getAddress);
```

Comment:
Wonder if it's worth console.log how many addresses were loaded from the exclusion list.